lib/api/controllers/authController.js

```diff
@@ -79,21 +79,24 @@ class AuthController {
     const strategy = request.input.args.strategy;
 
     return this.kuzzle.passport.authenticate({body: request.input.body, query: request.input.args, original: request}, strategy)
+      .then(userObject => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3524)